dasharo-compatibility/display-ports-and-lcd-support.robot

```diff
@@ -52,25 +52,25 @@ DSP002.201 - External HDMI display in OS (Ubuntu)
     [Documentation]    Check whether an external HDMI display is visible in
     ...    Linux OS. An external HDMI display must be provided in
     ...    the platform config.
-    ...    Previous IDs: DSP002.002
+    ...    Previous IDs: DSP002.001
```

Comment:
Where does this change come from? From what I can see in the database, it used to be .002, .001 is `Internal LCD in firmware`.

Reply:
Then it would mean that we have an error in the database. It was imported that way from old spreadsheet, I wonder at which point it changed :thinking: @macpijan CC, this needs verification.

Reply:
@macpijan ping, this blocks merging, which blocks few other tasks.